reference/constraints/Expression.rst

```diff
@@ -143,6 +143,11 @@ expression that must return true in order for validation to pass. To learn
 more about the expression language syntax, see
 :doc:`/components/expression_language/syntax`.
 
+.. versionadded:: 6.2
+
+    Alternatively, you can set the `negate`_ option to ``false`` in order to
+    assert that the expression must return true for validation to fail.
```

Comment:
```suggestion
    assert that the expression must return ``true`` for validation to fail.
```

Reply:
The whole block should be outside of the versionadded directive, otherwise it gets removed in Symfony 7 docs.

Versionadded directive should only contain:

The negate option was introduced in Symfony 6.2

This versionadded directive should also be added below the option in the other file